modules/user_data/templates/install_graphdb.sh.tpl

```diff
@@ -115,6 +100,7 @@ if [ "$graphdb_device: data" = "$(file -s $graphdb_device)" ]; then
 fi
 
 mount $graphdb_device /var/opt/graphdb/data
+chown -R graphdb:graphdb /var/opt/graphdb/data
```

Comment:
This should be already covered by the AMI setup script?

Reply:
needed because I think that after attaching the volumes this gets reverted to root:root, tested it